os/kernel/binary_manager/binary_manager.c

```diff
@@ -161,16 +185,12 @@ int binary_manager(int argc, char *argv[])
 		case BINMGR_UNREGISTER_STATECB:
 			binary_manager_unregister_statecb(request_msg.requester_pid);
 			break;
+#endif
 		default:
+			bmvdbg("Invalid cmd = %d\n", request_msg.cmd);
 			break;
 		}
 	}
 
-binary_manager_exit:
-	bmvdbg("Binary Manager EXITED\n");
-
-	mq_close(g_binmgr_mq_fd);
```

Comment:
Is it okay not to close and unlink the mq? I know there is no normal case that binary manager is terminated, but it can be happened because of something not common. In this case, there will be kernel resource leak.

Reply:
yes. right. but there is no case that a loop for request messages of  binary manager although binary manager is failed.